packages/foreman/rubygem-rubyipmi/rubygem-rubyipmi.spec

```diff
@@ -1,80 +1,91 @@
+# template: scl
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name rubyipmi
 
-Summary: A ruby wrapper for ipmi command line tools
 Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 0.10.0
-Release: 3%{?dist}
+Release: 4%{?dist}
+Summary: A ruby wrapper for ipmi command line tools that supports ipmitool and freeipmi
 Group: Development/Languages
-License: LGPLv2
-URL: https://github.com/logicminds/rubyipmi
+License: LGPLv2.1
```

Comment:
this was changed from 2.1 to 2 in https://github.com/theforeman/foreman-packaging/commit/7833c2d3f31ce9f1e00ad5cb67d7a5a05f641963#diff-a8eaae8f93e67347c371d8bdb1148e66, so your change would be a regression :)